src/extension/workspaceConfiguration.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import * as vscode from "vscode";
+
+import {ConfigurationReader} from "../common/configurationReader";
+import {Packager} from "../common/packager";
+
+export class WorkspaceConfiguration {
```

Comment:
We already have a `SettingsHelper` class. I would move the only method of this class there.